lib/fbe/octo.rb

```diff
@@ -691,61 +749,160 @@ def pull_request_review_comment_reactions(_name, _comment)
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_405,
+            name: 'shellcheck',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_379,
+            name: 'yamllint',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          }
+        ]
+      },
+      'zerocracy/judges-action' => {
+        total_count: 7,
+        check_runs: [
+          {
+            id: 28_907_016_501,
+            name: 'Codacy Static Code Analysis',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'codacy-production'
+            }
+          },
+          {
+            id: 28_906_596_603,
+            name: 'copyrights',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_550,
+            name: 'markdown-lint',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_483,
+            name: 'pdd',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_433,
+            name: 'rake',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_405,
+            name: 'shellcheck',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          },
+          {
+            id: 28_906_596_379,
+            name: 'yamllint',
+            head_sha: sha,
+            started_at: '2024-08-18T08:04:44Z',
+            completed_at: '2024-08-18T08:20:17Z',
+            app: {
+              slug: 'github-actions'
+            }
+          }
+        ]
+      }
     }
+    data[repo] || { total_count: 0, check_runs: [] }
```

Comment:
@Suban05 Maybe use `fetch` method, will be more clear?
```ruby
data.fetch(repo) do
  { total_count: 0, check_runs: [] }
end
```

Reply:
@Yegorov yes, thanks, I didn't think about it